src/components/Button/Button.tsx

```diff
@@ -16,7 +16,7 @@ const active = {
   transition: 'background 0s',
 }
 
-const StyledButton = styled(DEFAULT_TAG, {
+export const buttonBaseStyle = {
```

Comment:
Might need to be more separated.

Not sure destructive makes sense for the tabs.